storybook/src/Safei/components/Atoms/Toggles/Toggles.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Toggle from "./Toggle";
+import Checkbox from "./CheckBox";
+
```

Comment:
More tests needed for changing disabled status for example

Reply:
added more tests, as agreed I just covered a hand pick of cases since the variations are so many. 🙋